invenio/modules/deposit/testsuite/test_deposit_validators.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Unit tests for deposit module validators."""
+
+from invenio.testsuite import InvenioTestCase, make_test_suite, run_test_suite
+
+from wtforms.validators import ValidationError
+
+
+class Field(object):
+
+    def __init__(self, old_doi, new_doi):
+        self.object_data = old_doi
+        self.data = new_doi
+
+
+class Form(object):
+    pass
+
+
+ZENODO_DOI_PREFIX = "10.5072"
```

Comment:
Can you use e.g. ``LOCAL_DOI_PREFIX`` and ``REMOTE_DOI_PREFIX`` instead. Better to not have mentions of Zenodo in Invenio code.